apis/swagger.yml

```diff
@@ -423,6 +423,28 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"
 
+  /images/{imageid}/push:
+    post:
+      summary: "push an image on the registry"
+      description: "push an image on the registry"
+      consumes:
+        - "application/json"
```

Comment:
Here maybe we need some discussion. I think there the in HTTP request, we are not consuming `application/json`, and actually it is `application/octet-stream` that consumes type of file, like that described in moby: https://github.com/moby/moby/blob/master/api/swagger.yaml#L6735 .

Reply:
Updated